soc/silabs/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) 2017 Christian Taedcke
 
 add_subdirectory(common)
+zephyr_include_directories(${SOC_FAMILY})
 zephyr_include_directories(${SOC_FAMILY}/${SOC_SERIES})
```

Comment:
Did you check if `${SOC_FAMILY}/${SOC_SERIES}` has a higher precedence than `${SOC_FAMILY}`?

Reply:
It doesn't really matter, as files don't exist in both locations for the same series.